islands/BotTokenInput.tsx

```diff
@@ -0,0 +1,28 @@
+/** @jsx h  */
+import { h } from "preact";
+import { useEffect, useState } from "preact/hooks";
+import { tw } from "twind";
+
+export default function BotTokenInput() {
+  const [token, setToken] = useState("");
+
+  useEffect(() => {
+    setToken(localStorage.getItem("token") ?? "");
+  }, []);
+
+  useEffect(() => {
+    localStorage.setItem("token", token);
+  }, [token]);
+
+  return (
+    <input
+      type="password"
+      id="bot_token"
+      name="bot_token"
+      class={tw`bg-gray-50 border focus:outline-none border-gray-300 text-gray-900 rounded-lg focus:ring-grammy-500 focus:border-grammy-500 w-full p-2.5`}
+      placeholder="123456:ABCdef"
+      value={token}
+      onChange={(e) => setToken(e.currentTarget.value)}
```

Comment:
Would love if we could validate the bot token (getMe) with a little delay (something I didnt do in the cookie-version haha) and show red border for error. And green for success. And if it'll look good: with the getMe response, we could add a text below about the linked bot (show its bot token maybe).

This can be done later as well. Don't have to be in this PQ. What do you think?

Reply:
Done.